go/worker/compute/executor/committee/node.go

```diff
@@ -290,15 +360,20 @@ func (n *Node) queueBatchBlocking(
 	return n.handleExternalBatchLocked(batch, hdr)
 }
 
-// HandleBatchFromTransactionSchedulerLocked processes a batch from the transaction scheduler.
+// handleInternalBatchLocked processes a batch from the internal transaction scheduler.
 // Guarded by n.commonNode.CrossNode.
-func (n *Node) HandleBatchFromTransactionSchedulerLocked(
+func (n *Node) handleInternalBatchLocked(
 	batchSpanCtx opentracing.SpanContext,
 	ioRoot hash.Hash,
 	batch transaction.RawBatch,
 	txnSchedSig signature.Signature,
 	inputStorageSigs []signature.Signature,
-) {
+) error {
+	// If we are not waiting for a batch, don't do anything.
```

Comment:
Why is this needed now?

Reply:
indeed not needed anymore since we already do the check in `Dispatch`